RecoTracker/TkTrackingRegions/plugins/GlobalTrackingRegionWithVerticesProducer.h

```diff
@@ -128,6 +192,17 @@ class GlobalTrackingRegionWithVerticesProducer : public TrackingRegionProducer
   edm::EDGetTokenT<reco::VertexCollection> 	 token_vertex; 
   edm::EDGetTokenT<reco::BeamSpot> 	 token_beamSpot; 
 
+  //HI-related variables
+  edm::InputTag pixelClustersForScaling;
```

Comment:
this parameter is not necessary, all necessary reading functionality is contained in token_pc already